### PR TITLE
Default TMS on in CIME runs

### PR DESCRIPTION
**Pull request by tcclevenger**
Also adds an error message if trying to use TMS with Physics GLL.

@crterai Any interest in this going in the cess-production branch? Shouldn't change anything about the runs, could remove some lines in scripts I suspect.

### CODE REVIEW

File: components/eamxx/cime_config/namelist_defaults_scream.xml [hunk 1]
```diff
@@ -317,6 +319,9 @@ be lost if SCREAM_HACK_XML is not enabled.
       <cosp_frequency_units valid_values="steps,hours">hours</cosp_frequency_units>
     </cosp>
 
+    <!-- Turbulent Mountain Stress -->
+    <tms inherit="atm_proc_base"/>
```
Comment (bartgol) [inline components/eamxx/cime_config/namelist_defaults_scream.xml:323]: If this was not in master, how were noel/chris able to add tms via atmchange?

File: components/eamxx/cime_config/namelist_defaults_scream.xml [hunk 2]
```diff
@@ -141,7 +141,7 @@ be lost if SCREAM_HACK_XML is not enabled.
       so it must be of the form (a,b,...).
       NOTE: *CANNOT* be changed.
     -->
-    <!-- <atm_procs_list>(sc_import,homme,((shoc,cldFraction,spa,p3),rrtmgp,sc_export))</atm_procs_list> -->
+    <!-- <atm_procs_list>(sc_import,homme,((tms,shoc,cldFraction,spa,p3),rrtmgp,sc_export))</atm_procs_list> -->
```
Comment (bartgol) [inline components/eamxx/cime_config/namelist_defaults_scream.xml:144]: We should remove this line. It uses an old syntax, so it is misleading.

**Review by bartgol — CHANGES_REQUESTED** [review]

**Comment by ambrad** [comment]
We expect to see that all pg2 nightly tests will require rebaselining (on baseline machines), right?

**Comment by tcclevenger** [comment]
@ambrad Correct.

File: components/eamxx/cime_config/namelist_defaults_scream.xml (hunk 1 above)
Comment (tcclevenger) [reply components/eamxx/cime_config/namelist_defaults_scream.xml:323]: That is a good question. It doesn't look like on the cess-production branch it is included either.

Comment (tcclevenger) [reply components/eamxx/cime_config/namelist_defaults_scream.xml:322]: Looking back through the eamxx-simulation chat, I think that Noel encountered the error of no TMS registered and commented out `./atmchange physics::mac_aero_mic::atm_procs_list="tms,..."` so most likely running without TMS, and Oksana was most likely running with a version of the code before TMS was removing .

**Comment by ndkeen** [comment]
I would be ok with it going in as long as there is a clear way of turning it off in the script (if needed).

**Review by bartgol — APPROVED** [review]

**Comment by bartgol** [comment]
> I would be ok with it going in as long as there is a clear way of turning it off in the script (if needed).

I think this should work
```
./atmchange mac_aero_mic::atm_procs_list=shoc,cldFraction,p3
```

**Comment by crterai** [comment]
@tcclevenger - Yes, let's add this to the cess-production branch as well.

**Comment by tcclevenger** [comment]
@AaronDonahue DIFFs on Mappy expected, there is a random fail on weaver in DP unit tests which has nothing to do with this PR. If you are ok with it, feel free to merge. 

I'm going to disable DP tests in a PR since we are going a different route with that conversion. Don't want fails in those test to mess with AT anymore.

**Comment by bartgol** [comment]
@tcclevenger I say go ahead and merge.

**Comment by tcclevenger** [comment]
@bartgol @ambrad We'll need new baselines. I can take care of Mappy AT, do either of you have access to the others?

**Comment by bartgol** [comment]
We can't create baselines on Chrysalis until nightlies run. I don't think pm does baseline tests. I don't know about crusher/ascent, since they are not handled through the jenkins farms.

**Comment by bartgol** [comment]
> I'm going to disable DP tests in a PR since we are going a different route with that conversion. Don't want fails in those test to mess with AT anymore.

@tcclevenger are you still planning on doing this? All PRs are failing b/c of this test, so I'd like to disable it asap.

**Comment by tcclevenger** [comment]
@bartgol Yes, completely forgot to submit the PR. Will do it now.

**Pull request merged**